temporian/core/operators/calendar/base.py

```diff
@@ -56,9 +66,32 @@ def build_op_definition(cls) -> pb.OperatorDef:
         return pb.OperatorDef(
             key=cls.operator_def_key(),
             inputs=[pb.OperatorDef.Input(key="sampling")],
+            attributes=[
+                pb.OperatorDef.Attribute(
+                    key="utc_offset",
+                    type=pb.OperatorDef.Attribute.Type.FLOAT_64,
+                ),
+            ],
             outputs=[pb.OperatorDef.Output(key="output")],
         )
 
+    @staticmethod
+    def convert_to_utc_offset(timezone: Union[str, float, int]) -> float:
+        if isinstance(timezone, str):
+            tz_time = datetime.datetime.now(tz=pytz.timezone(timezone))
```

Comment:
would add test for the case where timezone isn't one of the existing options (and check the error message is decent, or catch it and throw our own if not)